kafka-connect/scripts/kafka_connect_run.sh

```diff
@@ -70,6 +70,17 @@ EOF
   # ... but enable equivalent GC logging to stdout
   export KAFKA_GC_LOG_OPTS="-verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps"
 
+  if [ -z "$KAFKA_CONNECT_LOG_LEVEL" ]; then
+    KAFKA_CONNECT_LOG_LEVEL="INFO"
+  fi
+  if [ -z "$KAFKA_LOG4J_OPTS" ]; then
```

Comment:
My point was : do we want the user to be able to use a completely different log4j properties file compared to the one we provide in our Kafka image ? If not, we should raising some warning or not starting Kafka at all if the `KAFKA_LOG4J_OPTS` is already set when here. Genuine question just to clarify what we want. It's also true what @scholzj said that the properties file can be replaced just with a volume.

Reply:
As per our conversation yesterday the bigger problem is that it's not possible to reconfigure the logging on the fly. And I don't really want to put effort into solving that when it seems likely Kafka will switch  from log4j to slf4j soon. So I feel this is good enough for now, and we can revisit it in the not too distant future.